helpers/query_params.go

```diff
@@ -0,0 +1,13 @@
+package helpers
+
+import "net/url"
+
+func GenerateURLParams(params map[string]interface{}) url.Values {
+	values := url.Values{}
+
+	for key, value := range params {
+		values.Add(key, value.(string))
+	}
+
+	return values
+}
```

Comment:
This file is used only by other tests. Should we move it under tests, in a subfolder like `test/helpers`? This way we can isolate "test utilities" from the code needed by the app 🤔